front/src/app/api/types.ts

```diff
@@ -149,13 +149,25 @@ export interface IBenchmarkMap {
     redshiftGpu: string;
 }
 
+export enum EOrderType {
+    ANY = 0,
```

Comment:
camelCase

also please start enum type with "Enum"  (like EnumProfileStatus, EnumProfileRole), not only E
interface definition starts With "I"(interface) like in c#
other definitions starts with "T" ("type")